src/content/docs/es/guides/dingo/002-quick-start-overview.md

```
@@ -1,390 +1,184 @@
-Para ver que puertos estan actualmente abiertos podemos ejecutar:
+> 💡 Consejo: UFW significa Uncomplicated Firewall y se usa para administrar reglas de firewall iptables (netfilter).
+
+Para ver qué puertos están actualmente abiertos:
 
 ```
 sudo ufw status numbered
 ```
 
-#### Agregar Puerto 3001 para Comunicacion Ouroboros Node to Node (NtN)
-Para sincronizar la cadena y pasar datos entre nodos necesitamos abrir el puerto 3001 o el puerto que hayas seleccionado. Para abrir el puerto 3001 ejecutaremos:
+#### Agregar puerto 3001 para comunicación Ouroboros Node to Node (NtN)
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)